meilisearch-http/src/helpers/meilisearch.rs

```diff
@@ -225,26 +166,24 @@ impl<'a> SearchBuilder<'a> {
 
         let start = Instant::now();
         let result = query_builder.query(reader, &self.query, self.offset..(self.offset + self.limit));
-        let (docs, nb_hits) = result.map_err(|e| Error::SearchDocuments(e.to_string()))?;
+        let (docs, nb_hits) = result.map_err(ResponseError::search_documents)?;
```

Comment:
ok, this is why you created setter... Yet, isn't it a bit overkill?

Reply:
Not only, look at the message above.